### ISSUE TITLE
Configuring lint levels when using `clippy-driver`

### ISSUE THREAD

**Issue by ojeda** [rust-lang/rust-clippy#6799]
Now that Clippy describes a way to be used on its own (see https://github.com/rust-lang/rust-clippy/pull/6782), it would be nice to have a way to customize the lints level outside of code.

That is, either having a way to pass the `-A`/`-W`/`-D`/`-F` flags (without `__CLIPPY_HACKERY__`), or picking them up from the configuration file (see https://github.com/rust-lang/rust-clippy/issues/6798).

**Comment by flip1995** [comment]
This should be possible already. Do you have an example where this doesn't work? I checked it locally and if worked as expected:

```rust
// main.rs
fn main() { let _ = 3.14; }
```

```
$ clippy-driver main.rs
error: approximate value of `f{32, 64}::consts::PI` found. Consider using it directly
 --> main.rs:2:13
  |
2 |     let _ = 3.14;
  |             ^^^^
  |
  = note: `#[deny(clippy::approx_constant)]` on by default
  = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#approx_constant

error: aborting due to previous error
```

vs

```
$ clippy-driver main.rs -Aclippy::approx_constant
$ echo $?
0
```

**Comment by ojeda** [comment]
Confirmed -- it works now. Not sure if something changed since February, or I was mistaken back then.

Thanks!
